libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/xirr/XirrDescriptor.ts

```diff
@@ -0,0 +1,62 @@
+// (C) 2021 GoodData Corporation
+
+import { IInsightDefinition } from "@gooddata/sdk-model";
+
+import {
+    ILayoutDescriptor,
+    IVisualizationSizeInfo,
+    IVisualizationDescriptor,
+    PluggableVisualizationFactory,
+} from "../../../interfaces/VisualizationDescriptor";
+
+import { PluggableXirr } from "./PluggableXirr";
+import { DASHBOARD_LAYOUT_DEFAULT_KPI_HEIGHT, MAX_VISUALIZATION_HEIGHT } from "../constants";
+
+export class XirrDescriptor implements IVisualizationDescriptor {
+    public getFactory(): PluggableVisualizationFactory {
```

Comment:
This could probably extend the HeadlineDescriptor? It should be practically the same as Headline.

Reply:
No Xirr is always small. Headline is small when only primary measure used, otherwise it is bigger

Reply:
Oh, ok, makes sense, thanks.